PROJECT_README.md

```diff
@@ -0,0 +1,20 @@
+<p align="center">
+  <strong>{PROJECT_NAME}</strong>
+</p>
+
+---
+
+## Getting Started
+
+### Requirements
+
+- Ruby `3.1.2`
+- Xcode `{TARGET_VERSION}+`
+
+### Install Dependencies
+
+- `bundle install`
+- `bundle exec arkana`
```

Comment:
@markgravity We should mention about adding variables to the `.env` file as well

Reply:
@suho updated at [ad0248f](https://github.com/nimblehq/ios-templates/pull/547/commits/ad0248f462e86e5d29a879b5017e92bcb1e23ad4)